ogr/ogrsf_frmts/ogrsf_frmts.dox

```diff
@@ -2392,6 +2398,75 @@ definition must be taken into account.
 @since OGR 1.9.0
 */
 
+/**
+\fn OGRErr OGRLayer::AlterGeomFieldDefn( int iGeomField, const OGRFieldDefn* poNewGeomFieldDefn, int nFlagsIn );
+
+\brief Alter the definition of an existing geometry field on a layer.
+
+You must use this to alter the definition of an existing geometry field of a real layer.
+Internally the OGRFeatureDefn for the layer will be updated
+to reflect the altered field.  Applications should never modify the OGRFeatureDefn
+used by a layer directly.
+
+Note that altering the SRS does *not* cause coordinate reprojection to occur:
+this is simply a modification of the layer metadata (correcting a wrong SRS
+definition)
+
+This method should not be called while there are feature objects in existence that
```

Comment:
This caveat makes the above note regarding reprojection redundant, but I'd still err for caution and leave it. For the same reason, I'm wondering if a note along the lines of "no modification to existing geometries will ever be performed, so this method cannot be used to eg promote single part geometries to their multipart equivalents".

Reply:
good idea. added